clang/include/clang/Basic/TargetInfo.h

```diff
@@ -1434,7 +1434,14 @@ class TargetInfo : public TransferrableTargetInfo,
     }
 
   public:
-    BranchProtectionInfo() = default;
+    BranchProtectionInfo() {
+      SignReturnAddr = LangOptions::SignReturnAddressScopeKind::None;
+      SignKey = LangOptions::SignReturnAddressKeyKind::AKey;
+      BranchTargetEnforcement = false;
+      BranchProtectionPAuthLR = false;
+      GuardedControlStack = false;
+    };
```

Comment:
```suggestion
    BranchProtectionInfo() :
      SignReturnAddr(LangOptions::SignReturnAddressScopeKind::None),
      SignKey(LangOptions::SignReturnAddressKeyKind::AKey),
      BranchTargetEnforcement(false),
      BranchProtectionPAuthLR(false),
      GuardedControlStack(false)
    {}
```
This is more likely to show up warnings if something gets missed or reordered.

Reply:
right, added and clang-formated.